apps/web/src/pages/index.tsx

```diff
@@ -1,9 +1,18 @@
 import type { NextPage } from 'next';
 import 'reflect-metadata';
-import { Sidebar } from '../components/Sidebar';
+import { Form, Header, PageTitle, Sidebar } from '@components';
 
 const Home: NextPage = () => {
-  return <Sidebar></Sidebar>;
+  return <>
+    <div className="flex overflow-x-hidden h-screen mr-auto">
+      <Sidebar />
+      <div className="w-full p-4 flex-col">
+        <Header />
+        <PageTitle title="Resource Planning" description="Planning resourcing plan based on skills and tasks within your unit."/>
```

Comment:
Think this design has been updated 

Reference -> https://www.figma.com/file/QuGyUBu1JEkNaGTb8ZfC3l/Care-mapping-design

Reply:
Updated to new design mockups

![image](https://user-images.githubusercontent.com/13221550/189445340-c742ee65-5679-4c37-8e0a-32e30eaa767f.png)